services/vision/mlvision/classifier.go

```diff
@@ -62,3 +68,17 @@ func attemptToBuildClassifier(mlm mlmodel.Service) (classification.Classifier, e
 		return classifications, nil
 	}, nil
 }
+
+func checkIfClassifierWorks(ctx context.Context, cf classification.Classifier) (classification.Classifier, error) {
+	if cf == nil {
+		return nil, errors.New("Nil classifier function")
+	}
+
+	img := &image.RGBA{}
```

Comment:
This initializes as a (0,0) dimensional image. It's quite an edge case and I wouldn't normally expect this to work. Have you tested this in an integration test? You should add tests to see if these functions behave as expected.

Reply:
I tested this with the model that I had been using earlier and it worked, I can write a test as well!

Reply:
I meant in that often there is a "resize" operations that gets done on images when they get fed into an ML model to make them comform to what the model expects. "resizing" an (0,0) image seems like it would have caused a problem, but I guess it didn't!